src/lib.rs

```diff
@@ -422,3 +414,21 @@ mod contract {
         }
     }
 }
+
+#[cfg(target_arch = "wasm32")]
+#[global_allocator]
+static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
+
+#[cfg(target_arch = "wasm32")]
+#[panic_handler]
+#[no_mangle]
+pub unsafe fn on_panic(_info: &::core::panic::PanicInfo) -> ! {
+    ::core::arch::wasm32::unreachable();
+}
+
+#[cfg(target_arch = "wasm32")]
+#[alloc_error_handler]
+#[no_mangle]
+pub unsafe fn on_alloc_error(_: core::alloc::Layout) -> ! {
+    ::core::arch::wasm32::unreachable();
+}
```

Comment:
Just a small nit, feels like this should be above the contract module.

Reply:
Done in https://github.com/aurora-is-near/aurora-engine/pull/103/commits/d14fe517cef05e41b22b85d32c6ff150fbeb7237